### PR TITLE
Build, test and install WiredTiger 2.4.1

### PR DESCRIPTION
**Pull request by alq666**
WiredTiger is a high-performance database, similar to LevelDB in its use cases. This installs the latest stable version (prior to their acquisition by MongoDB).

The main entry point is `wt`, a CLI. It is also compiled with python bindings.

### CODE REVIEW

File: Library/Formula/wiredtiger.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    system "./configure", "--with-builtins=snappy,zlib",
+                          "--with-python",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "wt", "create", "table:test"
+    system "wt", "drop", "table:test"
+  end
```
Comment (bfontaine) [inline Library/Formula/wiredtiger.rb]: It should be `system "#{bin}/wt"` instead of `system "wt"`.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by mikemcquaid** [comment]
Please fix the audit warnings too: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/27015/version=yosemite/testReport/junit/brew-test-bot/yosemite/audit_wiredtiger___strict/

**Comment by bfontaine** [comment]
Any update here?

**Comment by DomT4** [comment]
Presuming dead from lack of response to comments. Please feel free to fix-up various comments and resubmit the PR if you're still interested, but closing here for now. Thanks for the submission!

**Pull request closed without merging**